style.css

```diff
@@ -23,13 +36,88 @@ ul {
 	list-style: none;
 }
 
-a {
-	display: block;
-	text-align: center;
-	color: #a6a2c5;
-	text-transform: uppercase;
+.item-link {
 	padding: 15px;
 	text-decoration: none;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.item-icon {
+	margin-bottom: 5px;
+	opacity: 1;
+	transform: translateY(0px);
+	clip-path: polygon(0 0, 0 100%, 100% 100%,100% 0);
+	transition: transform, opacity, clip-path;
+	transition-duration: 800ms;
+	transition-timing-function: ease-in-out;
+	transition-delay: 200ms;
+}
+
+.item-text {
 	font-size: 25px;
+	color: var(--item-color);
+	font-family: sans-serif;
+	font-weight: bold;
+	position: absolute;
+	opacity: 0;
+	transform: translateY(0);
+	transition: transform, opacity;
+	transition-duration: 500ms;
+	transition-timing-function: ease-in-out;
+	transition-delay: 100ms;
+}
+
+.item-dot {
+	display: block;
+	width: 8px;
+	height: 8px;
+	background-color: var(--item-color);
+	border-radius: 50%;
+	opacity: 0;
+	transform: scale(0) translateY(0);
+	transition: transform, opacity;
+	transition-duration: 500ms;
+	transition-delay: 200ms;
+}
+
+.item-circle-explosion {
+	display: block;
+	width: 130px;
+	height: 130px;
+	border: 43px solid var(--icon-color);
+	background-color: transparent;
+	border-radius: 50%;
+	top: 4px;
+	position: absolute;
+	opacity: 0;
+	transform: scale(0);
+	transition: transform, opacity, border;
+	transition-delay: 40ms;
+	transition-duration: 140ms;
+	
+}
+
+.item-link.active .item-icon {
+	opacity: 0;
+	clip-path: polygon(0 0, 100% 0, 100% 100%);
```

Comment:
The clip-path animation is still broken. You can use firefox, their dev tools are awesome to play around with clip-path: (see that little icon in the screenshot).

Tip: Disable the JS click event and transitions while playing with devtools polygon edition, to avoid weird conflicts/delays.

![image](https://user-images.githubusercontent.com/14869087/106684719-e8de3380-65be-11eb-9cf0-73f54d0c2f7e.png)